cypress/e2e/tables.cy.js

```diff
@@ -1,8 +1,9 @@
 import { User } from '@nextcloud/cypress'
 describe('The Home Page', () => {
 	it('successfully loads', () => {
-		const user = new User('admin', 'admin')
-		cy.login(user)
+		cy.createRandomUser().then(user => {
+			cy.login(user)
+		})
```

Comment:
Might be nice to move this out of the actual test and rather into the beforeEach hook https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests#Hooks